### PR TITLE
Add Grafana alerts docs page

### LINKED ISSUE #718: Add docs for user customizable grafana alerts
#### Description

To start, it would be good to catch basic thing like spikes in latency, system-level pods being down, instance count at max cluster capacity, autoscaling events, API errors, etc.

### PR DESCRIPTION
**Pull request by RobertLucian**
Closes #718.

---

checklist:

- [x] run `make test` and `make lint`
- [x] update docs and add any new files to `summary.md` (view in [gitbook](https://cortex-labs.gitbook.io/staging/-MOmCGMADSRNQahK3Kox/) after merging)

### CODE REVIEW

**Review by deliahu — APPROVED** [review]

**Review by miguelvr — COMMENTED** [review]
It sounds like alerts only work with slack. You should refer that we are exemplifying with slack, while other methods are possible.

I think the step of creating a slack channel is not necessary (it is a requirement, not a step for setting up the alerts)

File: docs/clusters/observability/alerts.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Alerts
```
Comment (miguelvr) [inline docs/clusters/observability/alerts.md:1]: nit: Alerts → Alerting

File: docs/summary.md [hunk 2]
```diff
@@ -16,6 +16,7 @@
 * Observability
   * [Logging](clusters/observability/logging.md)
   * [Metrics](clusters/observability/metrics.md)
+  * [Alerts](clusters/observability/alerts.md)
```
Comment (miguelvr) [inline docs/summary.md:19]: nit: Alerts → Alerting

**Comment by RobertLucian** [comment]
@miguelvr it doesn't seem to sound like alerts only work with Slack. Check this out:

```text
Alerts can be configured for a variety of notification channels such as for Slack, Discord, Microsoft Teams, PagerDuty, Telegram and traditional webhooks. In this example, we'll use Slack
```

**Review by miguelvr — APPROVED** [review]

**Pull request merged**